### PR TITLE
feat: Custom properties for row(s).

### PR DESCRIPTION
**Pull request by bardosmisi**
### <strong>Pull Request</strong>

Fixes the row, expandable-row and tree-table-row parts of APM-266103.

#### Type of PR
Other (if none of the above apply)

### CODE REVIEW

File: libs/barista-components/table/src/row.scss [hunk 1]
```diff
@@ -24,10 +28,17 @@
 }
 
 :host-context(.dt-table-interactive-rows):hover {
-  border: solid 1px $gray-500;
+  border: solid 1px var(--interactive-border-color);
```
Comment (tomheller) [inline libs/barista-components/table/src/row.scss:31]: Hmm, this was gray-500 before and is now gray-100?
```suggestion
  border: solid 1px var(--interactive-border-color);
```

Comment (bardosmisi) [reply libs/barista-components/table/src/row.scss:31]: I am not sure how this happened, good catch.

**Review by tomheller — APPROVED** [review]

**Comment by tomheller** [comment]
When squashing this, please reword to a feature.

**Review by lukasholzer — APPROVED** [review]

**Review by lukasholzer — DISMISSED** [review]

**Comment by lukasholzer** [comment]
@bardosmisi please reword to `refactor(tree-table): Change styling to use css-custom properties.` We don't have a `feature` commit scope. You can use `feat` instead for features [comitlint.conif.js](https://github.com/dynatrace-oss/barista/blob/master/commitlint.config.js#L5). Should be caught by the commit lint have you commited with `--no-verify`?

But In my opinion, it is a refactor as it does not add additional functionality it refactors existing behaviour to a newer technology.

**Comment by tomheller** [comment]
> But In my opinion, it is a refactor as it does not add additional functionality it refactors existing behaviour to a newer technology.

I agree, but we added all of the refactor of custom properties as a feature to show up in the changelog. For consistency, I would stick to that.

**Comment by tomheller** [comment]
Reworded the commit to match the commit guidelines and the other custom property updates.

**Review by rowa-audil — APPROVED** [review]
LGTM

**Review by tomheller — APPROVED** [review]

**Pull request merged**